### PR TITLE
LPS-987654

### PR DESCRIPTION
**Pull request by ifrah-adnan**

### CODE REVIEW

**Comment by Youness-ELMO** [comment]
ci:test:sf

**Comment by Youness-ELMO** [comment]
ci:test:sf

**Comment by AkramYoussoufi** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+		Form.gotoAddField(
+			fieldPositionNumber = ${fieldPositionNumber2},
+			fieldType = ${fieldType2});
+
+		Form.editFieldLabelMetal(fieldValue = ${fieldValue2});
+
+		Form.save();
+	}
+
+	macro changeFieldsForm {
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro:31]: change or edit?

File: portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+			fieldValue = ${fieldValue1});
+
+		FormFields.editText(
+			fieldName = ${fieldName2},
+			fieldValue = ${fieldValue2});
+
+		FormPortlet.submit();
+
+		Form.submitAgain();
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro:103]: The macros are equals...

File: portal-web/test/functional/com/liferay/portalweb/paths/path/Testex.path [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	<td>//*[@id="nav_control_panel.configuration"]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CLOSE_SIMULATION_MENU</td>
+	<td>//button[@aria-label='Close']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ALERTE</td>
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/paths/path/Testex.path:24]: code in English, plz...

**Review by manoelcyreno — CHANGES_REQUESTED** [review]

**Comment by manoelcyreno** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/testcase/MyTestFile.testcase [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+		task ("Then Check the values defined on Height and Width") {
+			Click(locator1 = "ControlMenu#SIMULATION");
+
+			Click(
+				key_deviceType = "Custom",
+				locator1 = "ControlMenuPreviewPanel#PREVIEW_DATA_DEVICE_BUTTON");
+
+			AssertTextEquals(
+				locator1 = "Testex#ALERTE",
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/testcase/MyTestFile.testcase:58]: french?

**Review by manoelcyreno — CHANGES_REQUESTED** [review]

**Comment by manoelcyreno** [comment]
you make the fixes, but you didnt run your code, to see if it works, so plz, only send, if it is working...

**Comment by AkramYoussoufi** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/testcase/MyTestFile.testcase (hunk 4 above)
Comment (ifrah-adnan) [reply portal-web/test/functional/com/liferay/portalweb/tests/testcase/MyTestFile.testcase:58]: ok sorry i will fix

**Comment by AkramYoussoufi** [comment]
ci:test:sf

**Comment by ifrah-adnan** [comment]
ci:test:sf

**Comment by ifrah-adnan** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+	macro SetHeightAndWidth {
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro:91]: pay attention to the details ... the name of this method is out of the standards ...

File: portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+		Form.editFieldLabelMetal(fieldValue = ${fieldValue1});
+
+		Click(
+			key_fieldName = "Age",
+			locator1 = "FormFields#TEXT_FIELD");
+
+		Form.editFieldLabelMetal(fieldValue = ${fieldValue2});
+
+		Form.save();
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/macros/macro/Testex.macro:53]: You can convert it in only 1 method, and use it in your tests...

**Review by manoelcyreno — APPROVED** [review]
looks goo, but pay attention to the details, and the maintain code situations...

**Comment by manoelcyreno** [comment]
ci:close

**Pull request closed without merging**